Source/WebCore/platform/graphics/GraphicsContextGL.h

```diff
@@ -46,6 +46,14 @@
 #endif
 #endif
 
+inline constexpr size_t gcGLSpanDynamicExtent = std::numeric_limits<size_t>::max();
+
+template<typename T, size_t Extent = gcGLSpanDynamicExtent>
+class GCGLSpan;
+
+template<typename... Types>
+struct GCGLSpanTuple;
+
```

Comment:
This is super interesting. Perhaps this forward declaration belongs in GraphicsTypesGL.h?

Reply:
It was there and I removed it from `GraphicsTypesGL.h` to make that header as-light-as-possible.